app/controllers/kuroko2/api/job_definitions_controller.rb

```diff
@@ -0,0 +1,81 @@
+class Kuroko2::Api::JobDefinitionsController < Kuroko2::Api::ApplicationController
+  include Garage::RestfulActions
+
+  private
+
+  def require_resources
+    definitions = Kuroko2::JobDefinition.all
+    @resources = definitions.map {|definition| Kuroko2::Api::JobDefinitionResource.new(definition) }
+  end
+
+  def create_resource
+    definition = Kuroko2::JobDefinition.new(definition_params(params))
+    unless definition.api_allowed?
```

Comment:
ditto https://github.com/cookpad/kuroko2/pull/87/files#r135692164

Reply:
I have fixed it in 55b4da1.